### DISCUSSION TITLE
21.02编译后烧录360t7的WAN口正常，但是烧录我自己的7981开发板后wan不能获取上级路由器分配的IP地址， help！

### DISCUSSION THREAD

**Discussion by lastvariable** [immortalwrt/immortalwrt#1518] [Q&A]
各位大牛， immortalwrt21.02 克隆下来编译后烧录360t7的WAN口正常，但是烧录我自己的7981开发板后wan(就是port3)不能获取上级路由器分配的IP地址，但是port3改成lan口可以正常使用。  help！ 这是什么情况啊。

**Comment by lastvariable** [comment]
初步发现问题了，大概率是7531的port5没有接。

**Reply by XiaoLrrr** [reply]
接了port5还是没用呢,是zx zx7981pg吗
